### PR TITLE
Add random comment in Main.java

### PR DESCRIPTION
**Pull request by cordela**

### CODE REVIEW

File: src/main/java/frc/robot/Main.java [hunk 1]
```diff
@@ -7,20 +7,26 @@
...
-     * <p>If you change your main robot class, change the parameter type.
+     * <p>
+     * If you change your main robot class, change the parameter type.
      */
     public static void main(String... args) {
         RobotBase.startRobot(Robot::new);
     }
 }
+
+// Hi i'm here!!!
```
Comment (cordela) [inline src/main/java/frc/robot/Main.java:32]: This is a test comment

Comment (cordela) [reply src/main/java/frc/robot/Main.java:32]: It's very important

**Review by cordela — COMMENTED** [review]
This is a test comment

**Review by DogLoverPink — APPROVED** [review]
this stinks bc stink

**Comment by nstringham** [comment]
BTW the reason the formatting was upset in the first place is because there should always be a blank line at the end of every text file (Including .java files)

**Pull request merged**